modules/nf-core/custom/matrixfilter/meta.yml

```diff
@@ -42,29 +42,38 @@ output:
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
+          pattern: "*.filtered.tsv"
       - "*.filtered.tsv":
-          type: file
-          description: Filtered version of input matrix
+          type: map
+          description: |
+            Groovy Map containing information on experiment.
+            e.g. [ id:'test' ]
           pattern: "*.filtered.tsv"
   - tests:
       - meta:
           type: map
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
+          pattern: "*.tests.tsv"
       - "*.tests.tsv":
-          type: file
-          description: Boolean matrix with pass/ fail status for each test on each feature
+          type: map
+          description: |
+            Groovy Map containing information on experiment.
+            e.g. [ id:'test' ]
           pattern: "*.tests.tsv"
   - session_info:
       - meta:
           type: map
           description: |
             Groovy Map containing information on experiment.
             e.g. [ id:'test' ]
-      - R_sessionInfo.log:
-          type: file
-          description: Log file containing R session information
+          pattern: "*.log"
+      - "*R_sessionInfo.log":
+          type: map
+          description: |
+            Groovy Map containing information on experiment.
+            e.g. [ id:'test' ]
```

Comment:
I think the update was needed because we changed one of the outputs per my request. But why `--fix` now does these kind of replacements also seems off to me. @mashehu @mirpedrol what do you guys think? is that expected behavior and should we just manually revert these changes or is something wrong with tools?

Reply:
Changing the name of the output and using `*R_sessionInfo.log` is the right change but replacing the type and description is not! We saw the same with other modules, it must be a weird bug with the `--fix` command, you can revert this.
And the same for the other outputs where this happened, like `filtered.csv` and `tests.csv`